fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ isLoading: false });
```

Comment:
Be careful not to cancel your loading state until your fetch() actually finishes